web_src/js/features/repo-home.ts

```diff
@@ -7,141 +7,144 @@ import {fomanticQuery} from '../modules/fomantic/base.ts';
 const {appSubUrl} = window.config;
 
 export function initRepoTopicBar() {
-  const mgrBtn = document.querySelector('#manage_topic');
-  if (!mgrBtn) return;
+  const mgrBtns = document.querySelectorAll<HTMLButtonElement>('#manage_topic');
```

Comment:
It doesn't seem right, there should be only one `#manage_topic`

If there are many, then it can't use the same `id`

Reply:
citation also has this problem. (noticed just now, not fixed)
there will be two elements but only one will be displayed. (mobile view/desktop view)

Reply:
Done in 0af1d75664462c11a60ae77c791d983af8b4eb07